### PR TITLE
Add readme, updates and repairs

### PR DESCRIPTION
**Pull request by Spectre5**
added readme
changed to all relative imports
fixed stephabegg
fixed installation via "pip install ."
fixed summitpost
correction to test_shared
updated app.py to include a convenient test function for localhost

### CODE REVIEW

**Comment by Spectre5** [comment]
Added a fix for the summitpost finder

**Comment by Spectre5** [comment]
oops, reject those last two dealing with test_shared.py

**Comment by Spectre5** [comment]
corrected typo in the test, it passes now

**Comment by Spectre5** [comment]
Can also test with tox using:
`tox`
`source .tox/py36/bin/activate`
`trfind 48.51152 -121.05789 "Forbidden Peak"`

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Trip report search tool used by ClimbPlan, hosted at https://climbplan.com.
+
+# Contributing
+
+This is mostly a weekend project, Pull requests are welcome!
+
+## Wishlist
+
+Ideas are tracked using the (issues page)[https://github.com/thatneat/trfind/issues].
```
Comment (jason-curtis) [inline Readme.md]: markdown syntax for the link is backwards!

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Browser Testing
+* `python app.py`
+* visit [http://localhost:5000/test](http://localhost:5000/test)
+
+## Contributors
+
+* [thatneat](https://github.com/thatneat)
+* [jaimemarijke](https://github.com/jaimemarijke)
+* [priti-wright](https://github.com/priti-wright)
```
Comment (jason-curtis) [inline Readme.md]: looks like we need one more here!

File: trfind/find.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```
Comment (jason-curtis) [inline trfind/find.py]: not sure we need this shebang as the file doesn't do anything by itself, right?

File: trfind/finders/stephabegg.py [hunk 4]
```diff
@@ -10,14 +10,15 @@
...
-from trfind.models import TripReportSummary
+from ..models import TripReportSummary
 
 STEPH_ABEGG_SITE = 'Steph Abegg'
 
 def remove_mount(title_lowered):
     ''' Page titles incldue stuff like 'mt. stuart' so when we search in the title, remove 'Mount' from our peak name '''
     if title_lowered.startswith('mount'):
         return title_lowered[len('mount'):]
+    return title_lowered
```
Comment (jason-curtis) [inline trfind/finders/stephabegg.py:21]: oops! nice catch. maybe add a regression test?

File: Readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+## Setup and Running
+
+* Clone the github repo
+* `python3 -m venv env`
+* `source env/bin/activate`
+* `pip install -r requirements.txt`
+
+### CLI Testing
+* `pip install --no-deps .`
+* `trfind 48.51152 -121.05789 "Forbidden Peak"`
```
Comment (jason-curtis) [inline Readme.md]: you could also do #11 by documenting how to use `tox` for unit testing

File: Readme.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# trfind
```
Comment (jason-curtis) [inline Readme.md:1]: Thanks for filling this out!

File: trfind/finders/summitpost.py [hunk 7]
```diff
@@ -1,27 +1,47 @@
 import urllib
 from urllib.parse import urljoin, urlencode
+import requests
```
Comment (jason-curtis) [inline trfind/finders/summitpost.py:3]: ha, yeah, I guess as long as we have a dep on `requests` we might as well use it.

**Review by jason-curtis — CHANGES_REQUESTED** [review]
looks good! only minor comments.

**Comment by jason-curtis** [comment]
fixes #1

File: Readme.md (hunk 1 above)
Comment (Spectre5) [reply Readme.md]: fixed

File: Readme.md (hunk 2 above)
Comment (Spectre5) [reply Readme.md]: added

File: trfind/find.py (hunk 3 above)
Comment (Spectre5) [reply trfind/find.py]: force of habit, removed

File: trfind/finders/stephabegg.py (hunk 4 above)
Comment (Spectre5) [reply trfind/finders/stephabegg.py:21]: added a test to test_shared.py and it passes

**Comment by Spectre5** [comment]
I think I've addressed these points in the new commit

File: Readme.md (hunk 5 above)
Comment (Spectre5) [reply Readme.md]: added

**Review by jason-curtis — APPROVED** [review]

**Pull request merged**